lib/src/transaction/msg/v2/bank/v2.msgsend.ts

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable camelcase */
+import ow from 'ow';
+import { Msg } from '../../../../cosmos/v1beta1/types/msg';
+import { ICoin } from '../../../../coin/coin';
+import { v2 } from '../../ow.types';
+import { InitConfigurations, CroSDK } from '../../../../core/cro';
+import { AddressType, validateAddress } from '../../../../utils/address';
+import { CosmosMsg } from '../../cosmosMsg';
+import { COSMOS_MSG_TYPEURL } from '../../../common/constants/typeurl';
+import * as legacyAmino from '../../../../cosmos/amino';
+import { Network } from '../../../../network/network';
+
+export const msgSendV2 = function (config: InitConfigurations) {
+    return class MsgSend implements CosmosMsg {
```

Comment:
```suggestion
    return class MsgSend implements CosmosMsg {
```

should we update here to `MsgSendV2` for better readability?